tests/testthat/test-survival-helpers.R

```diff
@@ -0,0 +1,56 @@
+test_that('survival helpers', {
+  skip_if_not_installed("parsnip", minimum_version = "1.0.4.9001")
+  library(survival)
+  library(parsnip)
+
+  times <- seq(1, 100, length.out = 5)
+  times2 <- seq(100, 200, length.out = 5)
+  events <- c(1, 0, 1, 0, 1)
+
+  right_c <- Surv(times, events)
+  left_c <- Surv(times, events, type = "left")
+  intv_c <- Surv(times, times2, events, type = "interval")
+  count_c <- Surv(times, times2, events)
+
+  expect_true(parsnip:::.is_surv(right_c))
+  expect_false(parsnip:::.is_surv(1, fail = FALSE))
+  expect_snapshot_error(parsnip:::.is_surv(1))
+
```

Comment:
```suggestion

expect_snapshot(
  error = TRUE,
  parsnip:::.check_cens_type(left_c, fail = TRUE)
)
expect_snapshot(
  error = TRUE,
  parsnip:::.check_cens_type(left_c, type = c("right", "interval"), fail = TRUE)
)

```
in case we add pluralization